spec/requests/graphql/mutations/term/update_term_spec.rb

```diff
@@ -0,0 +1,195 @@
+      let(:term) do
+        FactoryBot.create(:external_term,
+                          vocabulary: vocabulary,
+                          custom_fields: { 'classification' => 'Horses' })
+      end
+      let(:variables) do
+        {
+          input: {
+            vocabularyStringKey: vocabulary.string_key,
+            uri: term.uri,
+            altLabels: ['Uni']
+          }
+        }
+      end
+      let(:expected_response) do
+        %(
+          {
+            "term": {
+              "uri": "http://id.worldcat.org/fast/1161301/",
+              "prefLabel": "Unicorns",
+              "altLabels": ["Uni"],
+              "authority": "fast",
+              "termType": "external",
+              "customFields": [
+                { "field": "classification", "value": "Horses" },
+                { "field": "harry_potter_reference", "value": null }
+              ]
+            }
+          }
+        )
+      end
+
+      before { graphql query, variables }
+
+      it 'updates alt_labels for term' do
+        term.reload
+        expect(term.alt_labels).to contain_exactly 'Uni'
+      end
+
+      it 'preserves custom fields' do
+        term.reload
+        expect(term.custom_fields).to match('classification' => 'Horses')
+      end
+
+      it 'returns updated term' do
+        expect(response.body).to be_json_eql(expected_response).at_path('data/updateTerm')
+      end
+    end
+
+    context 'when adding multiple alt_labels' do
+      let(:term) { FactoryBot.create(:external_term, vocabulary: vocabulary) }
+      let(:variables) do
+        {
+          input: {
+            vocabularyStringKey: vocabulary.string_key,
+            uri: term.uri,
+            altLabels: ['Uni', 'Horse with Horn']
+          }
+        }
+      end
+      let(:expected_response) do
+        %(
+          {
+            "term": {
+              "uri": "http://id.worldcat.org/fast/1161301/",
+              "prefLabel": "Unicorns",
+              "altLabels": ["Uni", "Horse with Horn"],
+              "authority": "fast",
+              "termType": "external",
+              "customFields": [
+                { "field": "classification", "value": null },
+                { "field": "harry_potter_reference", "value": true }
+              ]
+            }
+          }
+        )
+      end
+
+      before { graphql query, variables }
+
+      it 'adds alt_labels for term' do
+        term.reload
+        expect(term.alt_labels).to match_array ['Uni', 'Horse with Horn']
+      end
+
+      it 'returns updated term' do
+        expect(response.body).to be_json_eql(expected_response).at_path('data/updateTerm')
+      end
+    end
+
+    context 'when updating term_type' do
+      let(:term) { FactoryBot.create(:external_term, vocabulary: vocabulary) }
+      let(:variables) do
+        {
+          input: {
+            vocabularyStringKey: vocabulary.string_key,
+            uri: term.uri,
+            termType: 'local'
```

Comment:
Would the mutation be rejected altogether because termType is present here, and it isn't a registered argument in the UpdateTerm mutation?

Reply:
Yea, that sounds right. But what I am trying to make sure is that termType isn't overridden, what prevents that from happening isn't as important. Though this is a test that I converted from the REST endpoint so maybe its not necessary anymore?

Reply:
Yeah, I think it might not be necessary now.  In a way, this test is now just testing that any invalid argument doesn't end up modifying the term type.  I double checked and we have a `'cannot change term_type'` test in term_spec.rb, so I'm comfortable with removing this test.

Reply:
Okay, I will remove it.